tests/cpp/include/test_mkldnn.h

```diff
@@ -227,6 +227,7 @@ struct OpAttrs {
   nnvm::NodeAttrs attrs;
   std::vector<DispatchMode> dispatches;
   std::set<OpReqType> requests;
+  std::set<int> accept_dims;
```

Comment:
Is the accept_dims always sorted? If not, consider use unordered_set

Reply:
No, it need not be sorted. Updating to use unordered_set. Thanks!